signer/xpi/cose.go

```diff
@@ -51,7 +51,7 @@ func (s *XPISigner) generateCOSEKeyPair(coseAlg *cose.Algorithm) (eeKey crypto.P
 	case nil:
 		err = fmt.Errorf("Cannot generate private key for nil cose Algorithm")
 	case cose.PS256:
-		const size = 2048
+		size := s.issuerKey.(*rsa.PrivateKey).N.BitLen()
```

Comment:
I think you could end up with an ECDSA key in `s.issuerKey` that would make this panic

Reply:
:+1: 

https://github.com/mozilla-services/autograph/pull/76/commits/f9dc49baf186474bff9a1b5d0401f1609cbe4a0f